requirements.py

```diff
@@ -2,7 +2,7 @@
     "argparse",
     "numpy",
     "requests",
-    "pandas >=0.21.0",
+    "pandas>=0.21.0, <=0.23.4",
```

Comment:
is this still needed?

Reply:
this is how I believe it should be specified, but it seems to be ignored by AppVeyor.

Reply:
Ah yes, but do we still need to require <=0.23? #181 fixes this then? sorry to just trying to wrap my head around what is going on between the two prs

Reply:
Unfortunately, #181 only fixes the easy parts for now. I suggest to merge this in, then I'll focus on the remaining issues with pandas 0.24.0 (which are in the statistics module and related to a changed functioning of `combine_first()`, so unrelated to this PR).

Reply:
Ok, sounds good.